.github/workflows/weekly-cea.yml

```
@@ -0,0 +1,51 @@
+name: Weekly CEA CI
```

```suggestion
name: Weekly CEA builds
```
Technically this is not Continuous Integration :)

---

Right!